.devcontainer/Dockerfile

```
@@ -1,5 +1,6 @@
 ARG GO_VERSION=1.17
 ARG RUST_TOOLCHAIN=1.56.1
+ARG GOLANGCI_LINT=1.44.0
```

```suggestion

FROM docker.io/golang:${GO_VERSION}-bullseye as go
ARG GOLANGCI_LINT_VERSION=v1.44.0
RUN for p in \
```

Let's try to make this consistent with the other app versions like `YQ_VERSION` and `CARGO_DENY_VERSION`. `YQ_VERSION` has a `v` prefix.

`RUST_TOOLCHAIN` should really be called `RUST_VERSION` but we can do that in a followup.